### ISSUE TITLE
Base data authors

### ISSUE THREAD

**Issue by AlexDo1** [VForWaTer/metacatalog-entry-creator#1]
@mmaelicke, base data (license, variable, unit, author, keywords) is generated in this notebook with data from your local (?) metacatalog instance: https://github.com/VForWaTer/metacatalog-entry-creator/blob/1ee74e733dd68fe4f6d7d611b350a9c9df1f2eba/Update_base_data.ipynb

There are no 'default' authors created with `metacatalog init`, so authors in base data are not concurrent with authors in metacatalog instances different from your instance.

If we want to further develop `metacatalog-entry-creator`, we should keep this in mind.
We might solve this by creating base data from the [default data in metacatalog](https://github.com/VForWaTer/metacatalog/tree/master/metacatalog/data) (but then, no authors are available in the entry-creator) or, in the best case, by connecting to the "official" metacatalog instance.

**Comment by mmaelicke** [comment]
> @mmaelicke, base data (license, variable, unit, author, keywords) is generated in this notebook with data from your local (?) metacatalog instance: https://github.com/VForWaTer/metacatalog-entry-creator/blob/1ee74e733dd68fe4f6d7d611b350a9c9df1f2eba/Update_base_data.ipynb
> 

Yes! 

> There are no 'default' authors created with `metacatalog init`, so authors in base data are not concurrent with authors in metacatalog instances different from your instance.
> 

because that would make no sense at all. 

> If we want to further develop `metacatalog-entry-creator`, we should keep this in mind. We might solve this by creating base data from the [default data in metacatalog](https://github.com/VForWaTer/metacatalog/tree/master/metacatalog/data) (but then, no authors are available in the entry-creator) or, in the best case, by connecting to the "official" metacatalog instance.

The whole point of this repo is to showcase. I don't think that it will go into production. For that, it needs a real database connection. But you can connect any metacatalog instance you like in the notebook, then the streamlit app will reflect the state of that databse.

Thus, I would conclude, that this is not a bug and an enhancement is also not necessary. We just use the app to showcase, how the final upload logic should look like in V-FOR-WaTer...

**Comment by AlexDo1** [comment]
Yes, I just wanted to make sure that we keep this in mind, as `metacatalog-entry-creator` will only work with your database instance.
If this is just a showcase app we of course don't have to do anything about this.
